spec/our_rspec_test.rb

```diff
@@ -0,0 +1,56 @@
+require_relative '../ruby'
+
+describe Enumerable do
+  describe '#my_each' do
+    it 'Performs a block of code if given and returns the passed enumerable' do
+      expect([1, 2, 3].my_each { |x| 5 * x }).to eq([1, 2, 3])
+    end
+  end
```

Comment:
Great work adding this test. However, you have only tested for a positive scenario. What happens if no block is given? You should also test for the negative scenarios.

![commonsense (1)](https://user-images.githubusercontent.com/32361963/101760528-083f6f80-3aec-11eb-9208-67b72bb19eb1.png)

[pathwright link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)